{
    "body": "### Steps to reproduce the issue\r\n\r\n@victorusu @haampie \r\n\r\n```console\r\n$ ftn a.f90\r\n$ spack load cray-mpich/wyfi76i\r\n$ ftn a.f90\r\na.f90:2:4:\r\n\r\n use mpi\r\n    1\r\nFatal Error: Can't open module file 'mpi.mod' for reading at (1): No such file or directory\r\ncompilation terminated.\r\n$ unset C_INCLUDE_PATH\r\n$ ftn a.f90\r\n$\r\n```\r\n\r\nThis is just a minimal example, but my initial problem was inside an environment (without calling directly `spack load`)\r\n\r\n### Information on your system\r\n\r\n* **Spack:** 0.16.0-1199-bff4469d8f\r\n* **Python:** 3.8.2\r\n* **Platform:** cray-sles15-broadwell\r\n* **Concretizer:** original\r\n\r\n```yaml\r\npackages:\r\n  ...\r\n  cray-mpich:\r\n    buildable: false\r\n    externals:\r\n    - spec: cray-mpich@7.7.15\r\n      modules:\r\n      - cray-mpich/7.7.15\r\n```\r\n### Additional information\r\n\r\n### General information\r\n\r\n<!-- These boxes can be checked by replacing [ ] with [x] or by clicking them after submitting the issue. -->\r\n- [x] I have run `spack debug report` and reported the version of Spack/Python/Platform\r\n- [x] I have run `spack maintainers <name-of-the-package>` and @mentioned any maintainers\r\n- [ ] I have uploaded the build log and environment files\r\n- [x] I have searched the issues of this repo and believe this is not a duplicate\r\n",
    "user": "albestro",
    "url": "https://api.github.com/repos/spack/spack/issues/21696",
    "updated_at": "2021-05-18 07:20:02",
    "created_at": "2021-02-16 12:22:16",
    "closed_at": "2021-05-18 07:20:02",
    "state": "closed",
    "title": "Installation issue: cray-mpich",
    "number": 21696,
    "milestone": null,
    "labels": [
        "build-error"
    ],
    "id": 809265699,
    "html_url": "https://github.com/spack/spack/issues/21696",
    "assignees": [],
    "comments": 2
}